modules/installation-special-config-kmod.adoc

```diff
@@ -453,23 +378,50 @@ $ cd ../kvc-simple-kmod
 $ make install DESTDIR=${FAKEROOT}/usr/local CONFDIR=${FAKEROOT}/etc/
 ----
 
-. Get a tool called `filetranspiler` and dependent software:
+. Clone the fakeroot directory, replacing any symbolic links with copies of their targets:
 +
 [source,terminal]
 ----
-$ cd .. ; sudo yum install -y python3
-git clone https://github.com/ashcrow/filetranspiler.git
+$ cd .. && rm -rf kmod-tree && cp -Lpr ${FAKEROOT} kmod-tree
+----
+
+. Create a Butane config file, `99-simple-kmod.bu`, that embeds the kernel module tree and some additional configuration:
++
+[source,yaml]
+----
+variant: openshift
+version: 4.8.0
+metadata:
+  name: 99-simple-kmod
+  labels:
+    machineconfiguration.openshift.io/role: worker <1>
+storage:
+  trees:
+    - local: kmod-tree
+systemd:
+  units:
+    - name: require-kvc-simple-kmod.service <2>
+      enabled: true
+      contents: |
+        [Unit]
+        Requires=kmods-via-containers@simple-kmod.service
+
+        [Service]
+        Type=oneshot
+        ExecStart=/usr/bin/true
+
+        [Install]
+        WantedBy=multi-user.target
```

Comment:
I think now that [this bug](https://bugzilla.redhat.com/show_bug.cgi?id=1812972) is fixed (`systemd-239-45.el8.x86_64`) we can now just enable the user unit directly.

Reply:
Yep:

```
variant: fcos
version: 1.3.0
systemd:
  units:
    - name: echo@.service
      contents: |
        [Service]
        Type=oneshot
        ExecStart=/bin/echo %i
        RemainAfterExit=yes
        [Install]
        WantedBy=multi-user.target
    - name: echo@foo.service
      enabled: true
```

```
[core@qemu0 ~]$ sudo systemctl status echo@foo.service
● echo@foo.service
   Loaded: loaded (/etc/systemd/system/echo@.service; enabled; vendor preset: enabled)
   Active: active (exited) since Thu 2021-06-10 13:35:58 UTC; 15s ago
  Process: 1221 ExecStart=/bin/echo foo (code=exited, status=0/SUCCESS)
 Main PID: 1221 (code=exited, status=0/SUCCESS)

Jun 10 13:35:58 localhost systemd[1]: Starting echo@foo.service...
Jun 10 13:35:58 localhost echo[1221]: foo
Jun 10 13:35:58 localhost systemd[1]: Started echo@foo.service.
[core@qemu0 ~]$ cat /etc/redhat-release 
Red Hat Enterprise Linux CoreOS release 4.8
[core@qemu0 ~]$ rpm -q systemd
systemd-239-45.el8.x86_64
[core@qemu0 ~]$ 
[core@qemu0 ~]$ cat /etc/systemd/system-preset/20-ignition.preset 
enable echo@.service foo
disable kola-runext.service
```

Reply:
Awesome!  Updated.